prime-router/docs/universal-pipeline/README.md

```diff
@@ -1,24 +1,262 @@
 # Overview
+The Universal Pipeline is the core of what ReportStream is, and the two terms are sometimes used interchangeably. The Universal Pipeline is responsible for receiving data from a sender and sending it to one or more receivers. The pipeline was designed to be agnostic of:
+- **data formats** - At present, the pipeline can receive and send *HL7v2* and *FHIR* formats
+- **data types** - While presently only handling the routing of ELR (Laboratory Data), The pipeline was designed to be able to process the [six core data sources](https://www.cdc.gov/ophdst/public-health-data-strategy/phds-core-data-sources.html) as identified by PHDS:
+    - Case data
+    - Laboratory data
+    - Emergency department data
+    - Vital statistics data
+    - Immunization data
+    - Healthcare capacity and utilization data
 
-![Universal Pipeline](../assets/pipeline.png)
+The rest of this document aims to explore the high level architecture of the Universal Pipeline and how exactly it can meet the promises outlined above!
 
-## Data Pipeline through ReportStream
-([useful reference](https://docs.google.com/document/d/1srnzARL_OZDm_-Rm9GyJjVVXmtPdOWscSgcBGep3dnQ/edit))
-- Include graphic of the pipeline phases (Receive, Convert, Route, Translate, Batch, Send) as well as the full architecture diagram
-- Metadata Store
-- Blob Store - include a sub-section for each folder, especially the receive and ready folders.
-- Receiver’s server
-- Lineage Tracking
-  - Report lineage
-  - Item lineage
-- Asynchronous Pipeline (explain how this affects the history query)
-
-## Data Retention Policy
-
-## Pipeline steps
+## Data Pipeline
+In order to handle different data **formats** and **types** in a scalable and maintainable way, the pipeline has been broken up into the following independent "steps":
 
+```mermaid
+flowchart LR;
```

Comment:
This is a configuration option for the mermaid plugin. The flowchart LR tells mermaid to make a flow chart with the content below and orient it horizontally (LR) instead of the default of vertical